IABot/checkIfDead.php

```diff
@@ -58,23 +58,22 @@ public function checkDeadlinks( $urls ) {
 			curl_setopt( $curl_instances[$id], CURLOPT_NOBODY, 1 );
 			curl_setopt( $curl_instances[$id], CURLOPT_RETURNTRANSFER, true );
 			curl_setopt( $curl_instances[$id], CURLOPT_FOLLOWLOCATION, true );
-			curl_setopt( $curl_instances[$id], CURLOPT_TIMEOUT, 30 ); // Set 30 seconds timeout for the entire curl operation to take place
+			curl_setopt( $curl_instances[$id], CURLOPT_TIMEOUT, 60 ); // Set 60 second timeout for the entire curl operation to take place
```

Comment:
I don't follow.  This function only does a HEAD request, while a different function does the full request.  That seperate function doesn't even have a timeout which may result in a bot stall.  Wouldn't it be simpler to simply recurse this function where the recurse flag raises the timeout to 60 seconds and does a full GET requests on all failed URLs?